test/drenv/test.py

```diff
@@ -155,6 +156,7 @@ def enable_dr():
   namespace: {config['namespace']}
   annotations:
     drplacementcontrol.ramendr.openshift.io/is-cg-enabled: '{cg_enabled}'
+    drplacementcontrol.ramendr.openshift.io/do-not-delete-pvc: '{do_not_delete_pvc}'
```

Comment:
This is not correct, it should be added just before deleting the drpc, and the caller need to wait until the annotation is propagated to the vrgs. You can check how we do this in e2e:
https://github.com/RamenDR/ramen/blob/9f9340ba03e98d1efca6d1f0d53e714b59b3e42d/e2e/dractions/actions.go#L123

If this is correct for all ramen flows, ramen should add the annotation, or change to code to work as if the annotation was there.